### PR TITLE
Add tests that deal with applying number format specifiers on Symbols

### PR DESCRIPTION
**Pull request by domfarolino**
This PR is a follow-up to https://github.com/w3c/web-platform-tests/pull/9008 which now deals only with the application of the `%s` format specifier on Symbols.

This PR now includes tests for the application of `%i`, `%d`, and `%f` format specifiers on Symbols. The approval of this PR is pending on further discussion between implementations. At the time of creating, the spec and these tests mandate that Console implementations throw an Error when applying the aforementioned format specifiers on Symbols, though many browsers do not behave in this way currently.

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24241)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/340771709)

**Review by domenic — APPROVED** [review]
These tests LGTM and test the spec correctly, so I'm marking approved. But, we should probably delay merging until we have more clarity on the spec questions in https://github.com/whatwg/console/issues/113.

**Comment by domfarolino** [comment]
Sounds good, I've added a comment there to discuss two possible routes we could take with that part of the spec (one throwing on Symbol evaluation, the other not).

**Comment by domfarolino** [comment]
I've added tests to reflect the most recent spec PR following discussion in https://github.com/whatwg/console/issues/113. Since we've opted to not throw when applying %i/%d/%f format specifiers on Symbols, I've added a manual test instead of an an `-any.js` test, so testers can see the output `NaN` in the developer tools.

I'll merge the spec PR and, Domenic can review/merge this upon signing off when he's back!

File: console/console-number-format-specifiers-symbol-manual.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<body>
+<p>Open the console inside the developer tools. It should contain 15 lines, each of which are:</p>
+<p><code>NaN</code></p>
+
+<script>
+const methods = ["log", "dirxml", "trace", "group", "groupCollapsed"];
+
+for (method of methods) {
+  console[method]("%i", Symbol.for("description"));
+  if (method == "group" || method == "groupCollapsed") console.groupEnd();
```
Comment (domfarolino) [inline console/console-number-format-specifiers-symbol-manual.html]: This isn't the prettiest thing ever, but I'm kinda OK with it unless someone wants me to change it.

**Pull request merged**